app/scripts/lib/experiments/password-strength.js

```diff
@@ -15,24 +15,24 @@ class PasswordStrengthExperiment extends BaseExperiment {
   };
 
   onAccountCreated () {
-    this.saveState('account.created');
+    this.logEvent('account.created');
   }
 
   onPasswordError (error) {
     if (AuthErrors.is(error, 'PASSWORD_REQUIRED')) {
-      this.savePasswordBlockedState('missing');
+      this.logPasswordBlockedError('missing');
     } else if (AuthErrors.is(error, 'PASSWORD_TOO_SHORT')) {
-      this.savePasswordBlockedState('too_short');
+      this.logPasswordBlockedError('too_short');
     } else if (AuthErrors.is(error, 'PASSWORD_SAME_AS_EMAIL')) {
-      this.savePasswordBlockedState('email');
+      this.logPasswordBlockedError('email');
     } else if (AuthErrors.is(error, 'PASSWORD_TOO_COMMON')) {
-      this.savePasswordBlockedState('common');
+      this.logPasswordBlockedError('common');
     }
   }
 
-  savePasswordBlockedState (state) {
-    this.saveState('blocked');
-    this.saveState(state);
+  logPasswordBlockedError (state) {
+    this.logEvent('blocked');
```

Comment:
I changed from `saveState` to `logEvent` because `saveState` will allow at most one of each event type per browser, unless the user clears their localStorage.